source/PhysicalQuantity/Power.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace PhpUnitsOfMeasure\PhysicalQuantity;
+
+use PhpUnitsOfMeasure\AbstractPhysicalQuantity;
+use PhpUnitsOfMeasure\UnitOfMeasure;
+use PhpUnitsOfMeasure\HasSIUnitsTrait;
+
+class Power extends AbstractPhysicalQuantity
+{
+    use HasSIUnitsTrait;
+
+    protected static $unitDefinitions;
+
+    protected static function initialize()
+    {
+        // watt
+        $kiloWatt = UnitOfMeasure::nativeUnitFactory('W');
+        $kiloWatt->addAlias('watt');
+        $kiloWatt->addAlias('watts');
+        static::addUnit($kiloWatt);
+
+        static::addMissingSIPrefixedUnits(
+            $kiloWatt,
+            1,
+            '%pW',
+            [
+                '%Pwatt',
+                '%Pwatts',
+            ]
+        );
+
+        $metricHorsePower = UnitOfMeasure::linearUnitFactory('PS', 735.49875);
```

Comment:
This project has had frequent hard decisions about what to accept and what to reject when it comes to adding new quantities, units, and aliases.  I'm afraid that it would just be unworkable to include multiple languages worth of unit aliases with the way the code is currently implemented.

This does highlight the need for a change that I've been thinking more about recently; separating the definition of the quantities, units, and aliases from the code, with some sort of specification document.  I hadn't previously thought about it, but it might make sense to have localized versions of those specs.

For now, please remove the German unit alias.